repository-data/application/src/main/resources/hcm-config/hst/configurations/common/sitemap.yaml

```diff
@@ -10,4 +12,8 @@ definitions:
         jcr:primaryType: hst:sitemapitem
         hst:componentconfigurationid: hst:pages/search
         hst:refId: search
-        hst:relativecontentpath: ""
+        hst:relativecontentpath: publicationsfacet
+        /_any_:
+          jcr:primaryType: hst:sitemapitem
+          hst:componentconfigurationid: hst:pages/search
+          hst:relativecontentpath: publicationsfacet/${1}
```

Comment:
As you mentioned, we should rename this to something more generic, maybe `sitefacet`?